### ISSUE TITLE
Get user feedback channels update: Matrix issues?

### ISSUE THREAD

**Issue by danlatorre** [planetary-social/planetary-ios#1057]

**Comment by mplorentz** [comment]
One issue I have in the Matrix channel (which is bridged to Discord and Slack and Telegram) is that some messages don't seem to show up at all in the Matrix channel. I see people replying to messages that I can't see.

**Comment by danlatorre** [comment]
Yeah I'm just going to use the apps directly so I am in the same UX as users/testers. It's nice to have Slack get notifications for messages but I wish Slack told us where the messages are coming from, but for engagement/reply it's more clear to me to use Discord and Telegram directly. If there are not other issues, can we close this @rabble @mplorentz ?

**Comment by mplorentz** [comment]
If we aren't going to fix the matrix integration should we just turn it off? Or create a separate room that isn't bridged?
